packages/apps-config/src/endpoints/testing.ts

```diff
@@ -205,6 +205,13 @@ export function createTesting (t: TFunction): LinkOption[] {
         Interlay: 'wss://beta.polkabtc.io/api/parachain'
       }
     },
+    {
+      info: 'prism',
+      text: t('rpc.prism', 'Prism', { ns: 'apps-config' }),
```

Comment:
Is this a parachain (as per description) or a test chain? Either is fine, however if (as mentioned) it is a parachain, it needs to move to the testingRococo.ts file and have a paraId.

Reply:
Currently only a test chain